app/_src/kubernetes-ingress-controller/guides/high-availability/fallback-config.md

```diff
@@ -0,0 +1,327 @@
+- Automatically builds a fallback configuration that Kong will accept without requiring user intervention by
+  either:
+  - Excluding the broken objects along with its dependants.
+  - Backfilling the broken object along with its dependants using the last valid Kubernetes objects' in-memory cache (if `--use-last-valid-config-for-fallback` flag is set).
+- Enables users to inspect and identify what objects were excluded from or backfilled in the configuration using
+  diagnostic endpoints.
+
+Below table summarizes the behavior of the Fallback Configuration feature based on the configuration:
+
+| `FallbackConfiguration` feature gate value | `--use-last-valid-config-for-fallback` flag value | Behavior                                                                                                                                             |
+|--------------------------------------------|---------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `false`                                    | `false`/`true` (has no effect)                    | The last valid configuration is used as a whole to recover (if stored).                                                                              |
+| `true`                                     | `false`                                           | The Fallback Configuration is triggered - broken objects and their dependants are excluded.                                                          |
+| `true`                                     | `true`                                            | The Fallback Configuration is triggered - broken objects and their dependants are excluded and backfilled with their last valid version (if stored). |
+
+Below diagram illustrates how the Fallback Configuration feature works in detail:
+
+![fallback-config](/assets/images/products/kubernetes-ingress-controller/fallback-config.png "Fallback Configuration diagram")
+
+## Example Scenario
+
+In this example we'll demonstrate how the Fallback Configuration works in practice.
+
+{% include /md/kic/prerequisites.md release=page.release disable_gateway_api=false %}
+
+To test the Fallback Configuration, make sure your {{site.kic_product_name}} instance is running with
+the Fallback Configuration feature, diagnostics server enabled, and the Admission Webhook server disabled
+(that will enable us to slip a broken configuration to the controller):
+
+```bash
+helm upgrade --install kong kong/ingress -n kong \
+  --set ingressController.env.feature_gates=FallbackConfiguration=true \
+  --set ingressController.admissionWebhook.enabled=false
+```
+
+In the example, we'll consider a situation where:
+
+1. We have two `HTTPRoutes` pointing to the same `Service` and working correctly.
+2. One of the `HTTPRoutes` gets broken because of referencing an invalid `KongPlugin`.
```

Comment:
Nit: (I'm not sure what's the convention in the docs around this)

```suggestion
1. We have two `HTTPRoute`s pointing to the same `Service` and working correctly.
2. One of the `HTTPRoutes` gets broken because of referencing an invalid `KongPlugin`.
```